R/bind.r

```diff
@@ -1,90 +1,31 @@
-#' Efficiently bind multiple data frames by row and column
-#'
-#' This is an efficient implementation of the common pattern of
-#' `do.call(rbind, dfs)` or `do.call(cbind, dfs)` for binding many
-#' data frames into one.
-#'
-#' The output of `bind_rows()` will contain a column if that column
-#' appears in any of the inputs.
-#'
-#' @param ... Data frames to combine.
-#'
-#'   Each argument can either be a data frame, a list that could be a data
-#'   frame, or a list of data frames.
-#'
-#'   When row-binding, columns are matched by name, and any missing
-#'   columns will be filled with NA.
-#'
-#'   When column-binding, rows are matched by position, so all data
-#'   frames must have the same number of rows. To match by value, not
-#'   position, see [mutate-joins].
-#'
-#' @param .id The name of an optional identifier column.
-#'
-#'   If a string is supplied, the output will contain a column with that name
-#'   filled with identifiers that link each row to its original data frame. The
-#'   labels are taken from the named arguments to `bind_rows()`. When a list of
-#'   data frames is supplied, the labels are taken from the names of the list.
-#'   If no names are found, then a numeric sequence is used instead.
-#'
-#'   If `NULL`, the default, then no identifier column will be created.
-#'
-#' @param .name_repair One of `"unique"`, `"universal"`, or
-#'   `"check_unique"`. See [vctrs::vec_as_names()] for the meaning of these
-#'   options.
-#' @return `bind_rows()` and `bind_cols()` return the same type as
-#'   the first input, either a data frame, `tbl_df`, or `grouped_df`.
+#' Bind multiple data frames by row
+#'
+#' Bind any number of data frames by row, making a longer result. This is
+#' similar to `do.call(rbind, dfs)`, but the output will contain all columns
+#' that appear in any of the inputs.
+#'
+#' @param ... Data frames to combine. Each argument can either be a data frame,
+#'   a list that could be a data frame, or a list of data frames. Columns are
+#'   matched by name, and any missing columns will be filled with `NA`.
+#' @param .id The name of an optional identifier column. Provide a string to
+#'   create an output column that identifies each input. The column will use
+#'   names if available, otherwise it will use positions.
+#' @returns A data frame the same type as the first element of `...`.
```

Comment:
```suggestion
#' @return A data frame the same type as the first element of `...`.
```

I didn't know a `@returns` alias existed. Use whichever you prefer I guess, just calling it out in case it was an accident

Reply:
I think we prefer `@returns` generally, since it reads a little better.